bucket_CB/slop/specification

```diff
@@ -4,7 +4,7 @@ DEF[SOVERSION]=		${PORTVERSION}
 
 NAMEBASE=		slop
 VERSION=		${PORTVERSION}
-REVISION=		1
```

Comment:
technically bumping isn't required because we aren't changing the contents of the package or the run dependencies.  but it doesn't hurt.